packages/driver/cypress/e2e/commands/actions/click.cy.js

```diff
@@ -4103,10 +4103,10 @@ describe('mouse state', () => {
         const coordsChrome = {
           clientX: 492,
           clientY: 9,
-          layerX: 492,
-          layerY: 215,
           pageX: 492,
-          pageY: 215,
+          // pageY is 220.5 in headless Electron
+          // since updating to jquery 3.2+....why...
+          // pageY: 215,
```

Comment:
This `pageY` value returns 220.5 in headless Electron. It is 215 in headed, it is 215 in Chrome, it is 215 in the version of Chromium 118 that I could find. I commented this out. I have no earthly idea why a change in jQuery would result in this pageY value being slightly different when clicking from 1 div to another div in Electron headless.